app/routes/_auth+/login.tsx

```diff
@@ -116,18 +124,28 @@ export default function IndexLoginForm() {
           />
         </div>
 
-        <div>
+        <div className="relative">
           <Input
             label="Password"
             placeholder="**********"
             data-test-id="password"
             name={zo.fields.password()}
-            type="password"
+            type={showPassword ? "text" : "password"}
             autoComplete="new-password"
             disabled={disabled}
             inputClassName="w-full"
             error={zo.errors.password()?.message || data?.errors?.email}
           />
+          <span
```

Comment:
Please set the width of this span to 20px and the color to `text-gray-500`. Also when you resize make sure its properly aligned vertically.

Reply:
done